### DISCUSSION TITLE
Unable to apply for GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by wipalau** [community/community#81623] [GitHub Education]
### Hi everyone,

Hi everyone,
After the pandemic, when the physical classes started at our school started, all the email ID's were restricted from using the mail service, and on entering the name of my school GitHub asks for the email to be one of the domain my school owns. But I have the latest ID card of the school.

Please provide some assistance for my case.

**Comment by Shahnawazk7944** [comment]


**Reply by wipalau** [reply]
The school administrators are allowing for it because they have suspended their domain from being used for email, but It can only be used for Drive and Classroom apps.

**Reply by wipalau** [reply]
I need to add that email address in my GitHub account, But it has already expired, So I can't add that.

**Reply by Shahnawazk7944** [reply]
i don't get it, but still okay hope u'll get GSDP.
